lib/utils.pm

```diff
@@ -751,6 +753,8 @@ sub assert_and_click_until_screen_change {
     $repeat      //= 3;
     my $i = 0;
 
+    # FIXME: This is not totally race free - wait_screen_change may timeout, then the screen
+    # changes and the next assert_and_click will fail.
```

Comment:
Do you have an idea how to fix this? I could think of two solutions:
1. Ensure that we evaluate the very same screenshot for checking if the screen changed and then assert_and_click
2. In case of the screen initially matching but then wait_screen_change timing out *and* assert_and_click fails then continue, e.g. `eval` the `assert_and_click call (if this works from test context)

Reply:
I would go for (2.), but without resolving to `eval`, see os-autoinst/os-autoinst#1207.